share/p2p/peers/manager.go

```diff
@@ -66,6 +67,7 @@ type syncPool struct {
 
 	// isValidatedDataHash indicates if datahash was validated by receiving corresponding extended
 	// header from headerSub
+	isSynced            atomic.Bool
 	isValidatedDataHash atomic.Bool
```

Comment:
```suggestion
	// isValidatedDataHash indicates if datahash was validated by receiving corresponding extended
	// header from headerSub
	isValidatedDataHash atomic.Bool
	isSynced            atomic.Bool
```
match comment with field